src/components/Grid.jsx

```diff
@@ -26,7 +26,9 @@ class Grid extends React.Component {
     }
 
     return (n) => {
-      const pos = ((n + offset) / hours * 100) + '%'
+      const pos = ((n + offset) / hours * 100 + 0.1) + '%'
```

Comment:
You must be kidding me. This doesn’t shift just the first line, but all line with accumulated error (n * 0.1 %).